components/Layout/Layout.tsx

```diff
@@ -101,8 +102,12 @@ export default function AppLayout({ children }: AppLayoutProps) {
     }
   };
 
-  return presentPath.includes('/admin') && user.isAdmin ? (
-    <AdminLayout>{children}</AdminLayout>
+  return presentPath.includes('/admin') ? (
+    user.isAdmin ? (
+      <AdminLayout>{children}</AdminLayout>
+    ) : (
+      <AdminReject />
+    )
```

Comment:
관리자가 아니지만 접근하는 일반 유저면 기본적으로 관리자 layout이 아닌 기본 배경(우주 격자 배경)을 갖고있는게 UX적으로 적합하다고 생각이 들어요.
개선점으로는 
1. 경고 모달을 띄어주거나
2. 홈(`/`)으로 redirect하면서 toast 메세지로 접근 권한이 없음을 알림
해주면 좋을거 같아요.

1안같은 경우 잠깐 만져보니 아래 코드처럼 바꾸면 기본 배경에 경고 모달이 띄어지더군요

```suggestion
if (presentPath.includes('/admin')) {
    return user.isAdmin ? (
      <AdminLayout>{children}</AdminLayout>
    ) : (
      <div className={styles.appContainer}>
        <div className={styles.background}>
          <AdminReject />
        </div>
      </div>
    );
  }
```

Reply:
이미 배경 관련해서 CSS 적용해놓은상태라 1안대로 appcontainer에 배경 추가해놨습니다!
나중에 2안으로 개선해도 좋을것같아요